src/components/event/EventDetails.js

```diff
@@ -522,32 +511,43 @@ class EventDetails extends Component {
 			return (
 				<SafeAreaView style = { page }>
 					<NavBar title = { viewName } back onBack = { () => Actions.pop() } />
-					{ this.renderPickMembers() }
 					<View style = { container }>
 						<View style = { iconContainer }>
-							<Ionicons style = { [icon, textColor] } name = "md-calendar" size = { iconSize } color = '#000' />
+							<Ionicons
+								style = { [icon, textColor] }
+								name = "md-calendar" size = { iconSize }
+								color = "#000" />
 							<Text style = { [text, textColor] }>{ this.convertNumToDate(date) }</Text>
 						</View>
 						<View style = { iconContainer }>
-							<Ionicons style = { [icon, textColor] } name = "md-time" size = { iconSize } color = '#000' />
+							<Ionicons
+								style = { [icon, textColor] }
+								name = "md-time" size = { iconSize }
+								color = "#000" />
 							<Text style = { [text, textColor] }>{ startTime }-{ endTime }</Text>
 						</View>
 						<View style = { iconContainer }>
-							<Ionicons style = { [icon, textColor] } name = "md-pin" size = { iconSize } color = '#000' />
+							<Ionicons
+								style = { [icon, textColor] }
+								name = "md-pin" size = { iconSize }
+								color = "#000" />
 							<Text style = { [text, textColor] }>{ location }</Text>
 						</View>
 						{ description != ""
```

Comment:
Remove `!=`

Reply:
does an empty string evaluate to false?

Reply:
Yep, for reference: https://dorey.github.io/JavaScript-Equality-Table/